### DISCUSSION TITLE
If possible, continue to publish MV2 version until June 2023

### DISCUSSION THREAD

**Discussion by cpuuntery** [uBlockOrigin/uBlock-issues#2112] [Ideas]
So about #338
I know that is not an **everlasting** solution. But a temporary solution is **better than nothing**

So what I hope is there is a new file in the **GitHub releases** called 
uBlock0_X.XX.X.chromium MV2 policy Install.zip

and thanks to this file, it will be possible to use MV2 by say January 2023

the required policy is called **ExtensionInstallForcelist** 
[more](https://developer.chrome.com/docs/extensions/mv3/intro/mv3-migration/#modifying-network-requests) [info](https://chromeenterprise.google/policies/?policy=ExtensionInstallForcelist) [here ](https://admx.help/?Category=Chrome&Policy=Google.Policies.Chrome::ExtensionInstallForcelist)[[1]](https://developer.chrome.com/docs/extensions/mv3/intro/mv3-migration/#modifying-network-requests) [[2]](https://admx.help/?Category=Chrome&Policy=Google.Policies.Chrome::ExtensionInstallForcelist)

[![z-Xd-U3hdkj1-K0-Ks6t-Af-B4.png](https://i.postimg.cc/XqL5r6RX/z-Xd-U3hdkj1-K0-Ks6t-Af-B4.png)](https://postimg.cc/K3KjWwvy)

**Comment by u-RraaLL** [comment]
If you must stick to a Chromium engine, why not try another browser that uses it?

Configuring an Enterprise install for the sole purpose of running uBO on Chrome for a couple months longer seems like a waste of time to me.

**Comment by cpuuntery** [comment]
> If you must stick to a Chromium engine, why not try another browser that uses it?

Correct me if I am wrong. But as far as I understand and know. All other Chromium based browser will be affected.

> Configuring an Enterprise install for the sole purpose of running uBO on Chrome for a couple months longer seems like a waste of time to me.

Installing extension using a policy is not **rocket science** if you know your way around computers.

And it does not require you to **recompile Chromium to do it**.

I can not see the waste in it, the only not so positive thing about it that it is a **temporary solution**.

**Reply by u-RraaLL** [reply]
> Correct me if I am wrong. But as far as I understand and know. All other Chromium based browser will be affected.

Most(?) other Chromium based browsers said they won't be removing MV2 support, IIRC.

**Reply by TsukiZero** [reply]
From what I recall, Firefox doesn't plan to drop MV2 anytime soon, and they plan to do their own kind of MV3 to keep the good things of MV2. Haven't read this particular article, but it should answer:
https://blog.mozilla.org/addons/2022/05/18/manifest-v3-in-firefox-recap-next-steps/

**Reply by MasterKia** [reply]
> Most(?) other Chromium based browsers said they won't be removing MV2 support, IIRC.

Maybe, until Google decides to enforce a breaking change into Chromium that would make it impossible to keep MV2 compatibility.

Brave has its own built-in blocker (written in Rust and with uAssets compatibility in mind)

Vivaldi also has its own built-in blocker (similar to Adblock Plus, not as powerful as uBlock Origin).

Opera, I don't know.

Edge is going the direction Chromium is heading.



So in the end, if it becomes cumbersome to keep MV2 compatibility, IMO those Chromium-based browsers would remove it in a blink of an eye and resort to their own built-in blockers.

**Comment by krystian3w** [comment]
IMO nice will be compile these file as long MV2 works in latest version of forks like Opera, Brave, Edge.
